### PR TITLE
CRM457-1303: View application in caseworker after provider submission

### PR DESCRIPTION
**Pull request by patrick-laa**
Main changes, to make things fast and deterministic:
- Stop using workers
- Instead, process all background jobs synchronously
- Use explicit trigger for syncs
- **Add caseworker test steps that view application submitted by provider in previous test step**

I haven't yet worked out how to have a single test file that involves logging in as both provider and caseworker, as `test.use({ storageState: runTestAs('<role>') })` seems to only be allowed to be called once per `createTests` function. But that's ok, because by calling `submitClaimTests` before `viewSubmittedClaimTests` in `test.list.js`, we make sure one happens before t'other. When we have more of these tests we probably need naming conventions to make clear that a given caseworker test file has expectations pertaining to a given provider test file.

Also:
- Update to latest version of Playwright
- Remove various unused imports and excessive whitespace
- Standardise camelCase over snake_case in formData object.

### CODE REVIEW

**Review by dwhenry — APPROVED** [review]
don't really like the new test... but have no ideas for a better approach..

as this tests for know values, but doesn't ensure additional content doesn't exists... given this is the case does it make sense to check everything or could we slim the values we check to a subset instead of all the default value tests???? Not sure on this idea

**Comment by patrick-laa** [comment]
@dwhenry I know what you mean. I'm hoping @jogilder or @imtiazAhmd has a clearer sense of what we should be asserting in these caseworker tests to verify that what comes through is what the provider intended. I'm sort of thinking of this as a proof of concept of the technology, and assuming that the process of filling in different expectations in different scenarios comes later.

File: e2e/crm7/caseworker/view-submitted-claim.spec.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    let page;
+    test.describe.configure({ mode: 'serial' });
+
+    test.beforeAll(async ({ browser }) => {
+        page = await browser.newPage();
+    });
+    test.afterAll(async () => {
+        await page.close();
+    });
+    // Authenticate as provider
```
Comment (jogilder) [inline e2e/crm7/caseworker/view-submitted-claim.spec.js:17]: should say authenticate as caseworker

**Review by jogilder — COMMENTED** [review]
So I agree that I don't like this particular test and it is a little overkill but I guess making sure we have all the required information in the caseworker application is good. 

What it doesn't actually confirm is that this is the claim you created in the provider application. For this test I think it would would like to see the LAA reference number stored and then check that the same reference number appears in the list on caseworker.

**Comment by patrick-laa** [comment]
I've added a thing to pull the laaReference into a variable during the provider-side tests, then moved the caseworker-side tests into the same file to give them access to this variable to check that that reference is in the UI. This does mean moving away from the `test.use({ storageState: runTestAs('provider') })` paradigm for authentication, because that seems to assume that for one set of tests you're only signed in to one of the apps, and we want a batch of tests where you access both apps. But I've refactored to make it easy to do either.

**Review by jogilder — APPROVED** [review]
LGTM

**Pull request merged**